Exiled.Events/Patches/Generic/GhostMode.cs

```diff
@@ -232,7 +241,7 @@ private static bool Prefix(PlayerPositionManager __instance)
             }
             catch (Exception exception)
             {
-                Log.Error($"GhostMode error: {exception}");
+                Log.Error($"GhostMode error: {exception}\n{exception.InnerException}\n{exception.StackTrace}");
```

Comment:
Why? You literally print the same exception twice!1 Undo your change.

Reply:
Oops, leftover from testing something